library/spdm_requester_lib/libspdm_req_negotiate_algorithms.c

```diff
@@ -193,6 +193,12 @@ static libspdm_return_t libspdm_try_negotiate_algorithms(libspdm_context_t *spdm
         status = LIBSPDM_STATUS_INVALID_MSG_FIELD;
         goto receive_done;
     }
+    if (spdm_response->header.spdm_version >= SPDM_MESSAGE_VERSION_12) {
+        if (!libspdm_onehot0(spdm_response->other_params_support)) {
```

Comment:
The name of the response field should be `other_params_selection` but I'll file a separate issue for that.